### ISSUE TITLE
UMat.MinMax fails for some constant images and depth types

### ISSUE THREAD

**Issue by illiteratePhD** [emgucv/emgucv#174]
On commercial version 3.4.3

Running MinMax on an UMat only containing one unique value with depth types byte, short and ushort (might be others) returns min and max locations (-1, -1) and min and max values '0'. 

Example:
```
UMat uMat = new UMat(new Size(5, 5), DepthType.Cv8U, 1);
uMat.SetTo(new MCvScalar(255));
uMat.MinMax(out var minValues, out var maxValues, out var minLocations, out var maxLocations);
```
Observe results

**Comment by emgucv** [comment]
Tested and I can confirm this is an Open CV issue:

**Comment by emgucv** [comment]
Using a temporary fix to force running on CPU instead of OpenCL.

**Comment by emgucv** [comment]
Yes, it is an Open CV issue, need to be fixed on the Open CV repo side.

**Comment by emgucv** [comment]
This was a bug in OpenCV's OpenCL path for `minMaxLoc` on constant images. It was fixed in OpenCV and a CPU fallback workaround was applied on the Emgu CV side at the time. The bug no longer reproduces in current versions. Closing now.
